Project.toml

```diff
@@ -1,24 +1,26 @@
 name = "AxisSets"
 uuid = "a1a1544e-ba16-4f6d-8861-e833517b754e"
 authors = ["Invenia Technical Computing Corporation"]
-version = "0.1.6"
+version = "0.1.7"
 
 [deps]
 AutoHashEquals = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
 AxisKeys = "94b1ba4f-4ee9-5380-92f1-94cde586c3c5"
+FeatureTransforms = "8fd68953-04b8-4117-ac19-158bf6de9782"
 Impute = "f7bf1975-0170-51b9-8c5f-a992d46b9575"
 NamedDims = "356022a1-0364-5f58-8944-0da4b18d706f"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 ReadOnlyArrays = "988b38a3-91fc-5605-94a2-ee2116b3bd83"
 
 [compat]
 AutoHashEquals = "0.2"
-AxisKeys = "0.1"
+AxisKeys = "0.1.16"
+FeatureTransforms = "0.3.5"
 Impute = "0.6"
 NamedDims = "0.2"
 OrderedCollections = "1"
 ReadOnlyArrays = "0.1"
-julia = "1.3"
+julia = "1.5"
```

Comment:
We'll be able to revert this with https://github.com/invenia/FeatureTransforms.jl/pull/86

Reply:
Yeah, we should probably just wait for that to get merged first. Then we won't need to update the CI jobs either.